src/components/MultiHopTrade/components/TradeInput/components/Claim/ClaimStatus.tsx

```diff
@@ -79,53 +40,47 @@ export const ClaimStatus: React.FC<ClaimStatusProps> = ({
     return fromBaseUnit(activeClaim.amountCryptoBaseUnit, asset?.precision ?? 0)
   }, [asset, activeClaim])
 
-  const StatusBody = useMemo(() => {
+  const statusBody = useMemo(() => {
     if (!asset) return null
-
-    switch (claimTxStatus) {
-      case TxStatus.Pending: {
-        return (
-          <ClaimStatusBody
-            amountCryptoPrecision={amountCryptoPrecision}
-            description={translate('bridge.claimPending')}
-            // eslint-disable-next-line react-memo/require-usememo
-            icon={<CircularProgress size='24' />}
-            status={TxStatus.Pending}
-            symbol={asset.symbol}
-            title={translate('pools.waitingForConfirmation')}
-          />
-        )
-      }
-      case TxStatus.Confirmed: {
-        return (
-          <ClaimStatusBody
-            amountCryptoPrecision={amountCryptoPrecision}
-            description={translate('bridge.claimSuccess')}
-            // eslint-disable-next-line react-memo/require-usememo
-            icon={<CheckCircleIcon color='text.success' boxSize='24' />}
-            status={TxStatus.Confirmed}
-            symbol={asset.symbol}
-            title={translate('common.success')}
-          />
-        )
-      }
-      case TxStatus.Failed: {
-        return (
-          <ClaimStatusBody
-            amountCryptoPrecision={amountCryptoPrecision}
-            description={translate('bridge.claimFailed')}
-            // eslint-disable-next-line react-memo/require-usememo
-            icon={<WarningTwoIcon color='red.500' boxSize='24' />}
-            status={TxStatus.Failed}
-            symbol={asset.symbol}
-            title={translate('common.somethingWentWrong')}
-          />
-        )
+    const renderedAmount = (() => {
+      switch (claimTxStatus) {
+        case TxStatus.Pending: {
+          return (
+            <Amount.Crypto
+              prefix={translate('bridge.claimPending')}
+              value={amountCryptoPrecision}
+              symbol={asset.symbol}
+              color='text.subtle'
+            />
+          )
+        }
+        case TxStatus.Confirmed: {
+          return (
+            <Amount.Crypto
+              prefix={translate('bridge.claimSuccess')}
+              value={amountCryptoPrecision}
+              symbol={asset.symbol}
+              color='text.subtle'
+            />
+          )
+        }
+        case TxStatus.Failed: {
+          return (
+            <Amount.Crypto
+              prefix={translate('bridge.claimFailed')}
+              value={amountCryptoPrecision}
+              symbol={asset.symbol}
+              color='text.subtle'
+            />
+          )
+        }
+        case TxStatus.Unknown:
+        default:
+          return null
```

Comment:
suggestion: Most of the cases are only about changing the copies, we can remove the <Amount.Crypto />` occurrences and return the copy

Reply:
gm f411fefa05b6d5749f23b2945797d61782f50470